core/src/main/java/com/linecorp/armeria/common/SerializationFormat.java

```diff
@@ -94,7 +94,7 @@ private static SerializationFormat register(
             SerializationFormatProvider.Entry entry) {
 
         checkState(!uriTextToFormats.containsKey(entry.uriText),
-                   "serialization format registered already: ", entry.uriText);
+                   "serialization format registered already: %s", entry.uriText);
```

Comment:
😅

Reply:
Error Prone bugs have been fixed and detects our bugs. 😆